lib/hammer_cli_foreman/compute_profile.rb

```diff
@@ -0,0 +1,423 @@
+require 'hammer_cli_foreman/image'
+
+module HammerCLIForeman
+  class ComputeProfile < HammerCLIForeman::Command
+    resource :compute_profiles
+    def self.add_compute_resource(compute_resource)
+      @compute_resources ||= {}
+      @compute_resources[compute_resource.name] = compute_resource
+      @compute_resources
+    end
+
+    def self.compute_resources
+      @compute_resources
+    end
+    require 'hammer_cli_foreman/compute_resources/all'
+
+    def self.get_params(options)
+      params = {}
+      params['compute_attribute'] = {}
+      profile = HammerCLIForeman.record_to_common_format(
+        HammerCLIForeman.foreman_resource(:compute_profiles).call(:show, {'id' => options['option_compute_profile_id'] })
+      )
+      HammerCLIForeman.foreman_resource(:compute_resources).call(:show, {'id' => options['option_compute_resource_id'] })
+      params['compute_attribute'] = profile['compute_attributes'].select{ |hash|  hash['compute_resource_id']== options['option_compute_resource_id']}[0] || {}
```

Comment:
again, not important, but some spaces are missing ` .select{ ` and `hash['compute_resource_id']== `